### PR TITLE
Modify delivery list use case

### LINKED ISSUE #199: Update DG - Delivery List Use Case

### PR DESCRIPTION
**Pull request by juliusgambe**
Close #199

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1043,17 +1031,41 @@ otherwise)
...
+* 1e. User specifies both status and delivery date fields.
+    * 1e1. DMS displays a list of deliveries filtered by the specified status and delivery date.
+
+      Use Case Ends.
+
+* 1f. User specifies both customer and delivery date fields.
+    * 1f1. DMS displays a list of deliveries filtered by the specified customer and delivery date.
+
+      Use Case Ends.
+
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md]: Would there be a case of specifying status, customer, and delivery date fields?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -101,7 +101,8 @@ The `UI` component,
 
 ### Logic component
 
-**API** : [`Logic.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/logic/Logic.java)
+**
+API** : [`Logic.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/logic/Logic.java)
```
Comment (B-enguin) [inline docs/DeveloperGuide.md]: Perhaps there shouldn't be a newline

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -142,7 +143,8 @@ How the parsing works:
 
 ### Model component
 
-**API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
+**
+API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
```
Comment (B-enguin) [inline docs/DeveloperGuide.md]: Same as above

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -170,7 +172,8 @@ each `Person` needing their own `Tag` objects.<br>
 
 ### Storage component
 
-**API** : [`Storage.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/storage/Storage.java)
+**
+API** : [`Storage.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/storage/Storage.java)
```
Comment (B-enguin) [inline docs/DeveloperGuide.md]: Same as above

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (B-enguin) [reply docs/DeveloperGuide.md]: Also should these links be changed to our own team's repo

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -253,7 +242,7 @@ The following sequence diagram shows how the `login` command works:
 **Overview:**
 
 The `logout` command is used to log out from the user's account.
-Once logged out, the user will have no access to all the commands available, except for `help`, `exit`, 
+Once logged out, the user will have no access to all the commands available, except for `help`, `exit`,
 `register`, `login` and `delete account`.
```
Comment (B-enguin) [inline docs/DeveloperGuide.md]: Would the account recover command also be included in these cases

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -1321,31 +1319,56 @@ otherwise)
 **MSS:**
 
 1. Logged-in Owner types command to view a list of deliveries.
-2. DMS displays a list of all deliveries sorted by delivery date.
+2. DMS displays a list of all deliveries sorted in descending delivery date.
```
Comment (jianyangg) [inline docs/DeveloperGuide.md]: Perhaps we could specify descending in what sense
i.e., oldest to newest

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -100,7 +100,9 @@ The `UI` component,
 - depends on some classes in the `Model` component, as it displays `Person` object residing in the `Model`.
 
 ### Logic component
-**API** : [`Logic.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/logic/Logic.java)
+
+**
+API** : [`Logic.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/logic/Logic.java)
```
Comment (jianyangg) [inline docs/DeveloperGuide.md]: Formatting issues due to IDE

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -141,7 +143,8 @@ How the parsing works:
 
 ### Model component
 
-**API** : [`Model.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/model/Model.java)
+**
+API** : [`Model.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/model/Model.java)
```
Comment (jianyangg) [inline docs/DeveloperGuide.md]: Same as above

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -191,21 +193,21 @@ The `Customer` model,
 
 ### Storage component
 
-**API** : [`Storage.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/storage/Storage.java)
-
+**
+API** : [`Storage.java`](https://github.com/AY2324S1-CS2103T-T13-3/tp/tree/master/src/main/java/seedu/address/storage/Storage.java)
```
Comment (jianyangg) [inline docs/DeveloperGuide.md]: Same as above

File: docs/DeveloperGuide.md (hunk 7 above)
Comment (jianyangg) [reply docs/DeveloperGuide.md]: Might help to use the online web editor

**Review by jianyangg — APPROVED** [review]
Few nits to fix, otherwise LGTM!

**Pull request merged**